ui-tests/src/test/java/org/openmrs/reference/page/HeaderPage.java

```diff
@@ -3,33 +3,37 @@
 import org.openmrs.uitestframework.page.Page;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 public class HeaderPage extends Page {
 
     static final String HOME_ICON = "logo";
     static final String HOME_LINK_TEXT = "Home";
     static final By LOGOUT = By.className("logout");
+
     public HeaderPage(WebDriver driver) {
-        super(driver);
+	super(driver);
     }
 
     public void clickOnHomeIcon() throws InterruptedException {
-        clickOn(By.className(HOME_ICON));
+	clickOn(By.className(HOME_ICON));
     }
 
-
     // TODO This is unused, do we really need it?
     public void clickOnHomeLink() {
-        clickOn(By.linkText(HOME_LINK_TEXT));
+	clickOn(By.linkText(HOME_LINK_TEXT));
     }
 
     public void logOut() throws InterruptedException {
-        clickOn(LOGOUT);
+	waiter.until(ExpectedConditions.visibilityOfElementLocated(LOGOUT));
```

Comment:
Hello @k-joseph  i added this waiter method to resolve issues with element invisibility